filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log-expected.json

```diff
@@ -74,20 +74,16 @@
         "elasticsearch.slowlog.id": "s01HZ2QBk9jw4gtgaFtn",
         "elasticsearch.slowlog.logger": "index.indexing.slowlog.index",
         "elasticsearch.slowlog.routing": "",
-        "elasticsearch.slowlog.source_query": "\n{\n  \"@timestamp\":\"2018-07-04T21:27:30.730Z\",\n  \"metricset\":{\n    \"name\":\"network\",\n    \"module\":\"system\",\n    \"rtt\":7264},\n    \"system\":{\n      \"network\":{\n        \"name\":\"lo0\",\n        \"in\":{\n          \"errors\":0,\n          \"dropped\":0,\n          \"bytes\":77666873,\n          \"packets\":244595},\n          \"out\":{\n            \"packets\":244595,\n            \"bytes\":77666873,\n            \"errors\":0,\n            \"dropped\":0\n          }\n        }\n      },\n      \"beat\":{\n        \"name\":\"Rados-MacBook-Pro.local\",\n        \"hostname\":\"Rados-MacBook-Pro.local\",\n        \"version\":\"6.3.0\"\n      },\n      \"host\":{\n        \"name\":\"Rados-MacBook-Pro.local\"\n      }\n    }",
```

Comment:
Hi @pgomulka, can we ensure in 6.7/7.0 onwards that **plaintext** slow logs don't have multi-line un-escaped JSON in them like in this sample log entry's `source[...]` field:

https://github.com/elastic/beats/blob/35abb5fcbd5b5b594a6534b0765789bd680d287e/filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log#L4-L35

Instead, can we ensure that all **plain** slow logs have JSON in them that is on a single line like in this sample log entry's `source[...]` field:

https://github.com/elastic/beats/blob/35abb5fcbd5b5b594a6534b0765789bd680d287e/filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log#L3

It's not valid JSON anyway to have unescaped newlines in it. And, in the context of this PR, it means Filebeat cannot distinguish between a structured log entry (the work that you did) vs. a JSON object that's part of an unstructured log entry's `source[...]` field.

Reply:
If we don't want to remove that setting, another option would be to start the JSON object within the `source` field on the same line as the `source` field, so something like this:

   ```
   ... rest of log line ... source[{
     ...
     }]
   ```

or this:

   ```
   ... rest of log line ...
   source[{
     }]
   ```

Reply:
Thanks @pgomulka for addressing this with https://github.com/elastic/elasticsearch/pull/38081.